wazuh/wazuh-packages
# Remove demo certificates from indexer and dashboard packages

|Related issue|
|---|
| Closes https://github.com/wazuh/wazuh-packages/issues/1334 |


## Description

This PR removes the demo certificates from the indexer and dashboard packages. It also modifies the unattended to configure the certificates that are created because they do not exist previously and adjusts their permissions.

- When merged, the base tar and the dashboard and indexer packages will be updated

## Logs example

- AIO centos7 https://devel.ci.wazuh.info/view/Tests/job/Test_unattended/491/
- AIO ubuntu xenial https://devel.ci.wazuh.info/view/Tests/job/Test_unattended/490/
- Distributed https://devel.ci.wazuh.info/view/Tests/job/Test_unattended_distributed/385/

## Tests

- Related to jenkins
- See https://github.com/wazuh/wazuh-packages/issues/1334#issuecomment-1076665951


- Build the package in any supported platform
- [x] Package installation
- [x] Package remove
- [x] Package install/remove/install
- [x] Change added to CHANGELOG.md


- Tests for Linux RPM
  - [x] Build the package for x86_64
  - [x] `%files` section is correctly updated if necessary
- Tests for Linux deb
  - [x] Build the package for x86_64
  - [x] Package install/remove/install
  - [x] Package install/purge/install
  - [x] Check file permissions after installing the package
